js/src/options.ts

```diff
@@ -9,6 +9,7 @@ export type Options = {
     onExit?: () => any;        // Executes when the Immersive Reader exits
     customDomain?: string;     // Reserved for internal use. Custom domain where the Immersive Reader webapp is hosted (default is null).
     allowFullscreen?: boolean; // The ability to toggle fullscreen (default is true).
+    parent?: any;               // Node in which the iframe/webview container is placed (default is body).
```

Comment:
nit: Do we want to consider the type Node for the parent property? this will enforce passing in something which has appendChild/removeChild.